plugins/package-curation-providers/clearly-defined/src/funTest/kotlin/ClearlyDefinedPackageCurationProviderFunTest.kt

```diff
@@ -91,21 +105,33 @@ class ClearlyDefinedPackageCurationProviderFunTest : WordSpec({
             // Use an id which is known to have non-empty results from an earlier test.
             val packages = createPackagesFromIds("Maven:org.slf4j:slf4j-log4j12:1.7.30")
 
-            val curations = provider.getCurationsFor(packages)
+            withRetry {
+                val curations = provider.getCurationsFor(packages)
 
-            curations should beEmpty()
+                curations should beEmpty()
+            }
         }
 
         "be retrieved for packages without a namespace" {
             val provider = ClearlyDefinedPackageCurationProvider()
             val packages = createPackagesFromIds("NPM::acorn:0.6.0")
 
-            val curations = provider.getCurationsFor(packages)
+            withRetry {
+                val curations = provider.getCurationsFor(packages)
 
-            curations shouldNot beEmpty()
+                curations shouldNot beEmpty()
+            }
         }
     }
 })
 
 private fun createPackagesFromIds(vararg ids: String) =
     ids.map { Package.EMPTY.copy(id = Identifier(it)) }
+
+private suspend fun <T> withRetry(f: suspend () -> T): T =
+    retry(
+        maxRetry = 3,
+        timeout = 9.seconds,
+        delay = 3.seconds,
```

Comment:
I wonder whether instead of a fixed delay we should rather make use of `multiplier = 5` (or so; and increase the `timeout` accordingly), because *if* there is a continuous problem, it probably does not disappear after a too short time.

Reply:
I would like to try this one out first. The first two runs worked. We can change it later as you propose if it turns out not to work. A believe as-is it's faster.

Reply:
We still get the timeouts: https://github.com/oss-review-toolkit/ort/actions/runs/5308587525/jobs/9608290182#step:5:825

So I'll prepare a PR that changes to using a multiplier.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/7149.